action.yml

```
@@ -80,7 +79,7 @@ outputs:
       Version used to crate release and tag image.
 runs:
   using: docker
-  image: docker://europe-docker.pkg.dev/nube-hub/docker-public/velo-action:0.2.72
+  image: docker://europe-docker.pkg.dev/nube-artifacts-prod/nube-container-images-public/velo-action:814516a
```

That artifact registry is deprecated
use this 

```suggestion
  image: docker://europe-docker.pkg.dev/nube-hub/docker/velo-action:814516a
```

---

Oh, I thought it is the other way around. Where is the source of truth on that?